packages/ui/src/utils/NumberUtil.ts

```diff
@@ -0,0 +1,23 @@
+/**
```

Comment:
any way to test this? maybe already covered by the controller test?

Reply:
Done!